src/components/ui/form.js

```diff
@@ -17,96 +18,151 @@ export const Form = ({ pageTitle, conversionPageUrl }) => {
 
   const validationSchema = Yup.object().shape({
     firstName: Yup.string().required('Please enter your first name'),
-    dropdownField: Yup.string(),
+    lastName: Yup.string().required('Please enter your last name'),
+    emailAddress: Yup.string().required('Please enter your email address'),
+    phoneNumber: Yup.string(),
+    aboutYourDepartment: Yup.string().required('Please select one of the options'),
+    hobbiesChk: Yup.array(),
+    message: Yup.string().required('Please enter your message'),
   });
 
-  const formOptions = { resolver: yupResolver(validationSchema) };
-  const { register, formState, handleSubmit } = useForm(formOptions);
+  const { formState, register, handleSubmit, setValue, control } = useForm({
+    resolver: yupResolver(validationSchema),
+  });
   const { errors } = formState;
 
   const onSubmit = data => {
-    setLoading(true);
-    const submitURL = `/api/form-handler`;
+    console.log(data);
+    // setLoading(true);
+    // const submitURL = `/api/form-handler`;
+    // if (data) {
+    //   const webhookData = {
+    //     firstName: data.firstName,
+    //     dropdown: data.dropdownField,
+    //     conversionPageUrl: conversionPageUrl,
+    //     conversionPageTitle: pageTitle,
+    //   };
+    //   axios
+    //     .post(submitURL, webhookData)
+    //     .then(res => {
+    //       setIsSubmitted(true);
+    //       router.push('/thank-you' + conversionPageUrl);
+    //     })
+    //     .catch(err => {
+    //       console.error(err);
+    //       alert('There was an error submitting your form. Please try again.');
+    //     });
+    // }
+  };
```

Comment:
I noticed most of the `onSubmit` is commented out, is this intended? We might want to consider including it, but in a way that doesn't actually submit the data if that makes sense.